File: Livro Augusto Manzano/Capítulo 5/B5.rb
=begin
**********************************************************************************
*   File:    B5.rb                                                               *
*   Author:  Juan R. Rodrigues                                                   *                                     
*   Book: Algoritmos: Lógica para desenvolvimento de programação de computadores *
**********************************************************************************

Elaborar um programa que mostre os resultados da tabuada de um número qualquer, a qual
deve ser apresentada de acordo com sua forma tradicional.
 

=end
puts "Tabuada Tradicional de Multiplicação"
puts "Informe um numero inteiro"
numero = gets.to_i

sleep 1

produto = 0
contador = 0

loop do 
   contador += 1
   produto = (contador * numero)
   puts "#{contador}x#{numero}=#{produto}"
if contador == 10
      break
    end 
end  

File: Livro Augusto Manzano/Capítulo 5/C/B5.c
/*********************************************************************************
*   File:    B5.c                                                              *
*   Author:  Juan R. Rodrigues                                                   *                                     
*   Book: Algoritmos: Lógica para desenvolvimento de programação de computadores *
**********************************************************************************

Elaborar um programa que mostre os resultados da tabuada de um número qualquer, a qual
deve ser apresentada de acordo com sua forma tradicional. */
 
#include <stdio.h>

 int main(){

  int numero, produto, count;

  printf("Tabuada Tradicional de Multiplicação\n");
   printf("Informe um numero inteiro: "); 
    scanf("%d", &numero);


for(count=1; count <= 10;count++){
   produto  = count * numero;
   printf("\n%dx%d = %d", count, numero, produto);
   }
}


